modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/healthcheck/ping/PingHealthCheckConsensusDecision.scala

```diff
@@ -0,0 +1,7 @@
+package org.tessellation.sdk.infrastructure.healthcheck.ping
+
+import org.tessellation.sdk.domain.healthcheck.consensus.types.HealthCheckConsensusDecision
+
+final case class DecisionKeepPeer() extends HealthCheckConsensusDecision
```

Comment:
`case object` maybe instead of a `case class`? Then it would simplify the pattern-match even more
```
case (DecitionKeepPeer, round) =>
```